airflow/contrib/hooks/winrm_hook.py

```diff
@@ -204,6 +203,7 @@ def get_conn(self):
 
         try:
             if self.password and self.password.strip():
+                # pylint: disable=unexpected-keyword-arg
```

Comment:
Unrelated changes?

Reply:
> > This is exactly what I can do using --copy-from as described above
> 
> Are you thinking something like
> 
> ```dockerfile
> FROM base as compiler
> 
> RUN apt-get install build-essential mysql-dev ...
> 
> RUN pip install --local -e  '.[$extras]' && pip uninstall --yes apache-airflow
> 
> FROM base as prod
> 
> RUN apt-get install libmysql
> 
> COPY --from=compiler /home/airflow/.local /home/airflow/.local
> ```
> 
> Or have I misunderstood you?

Yep. very close to it.

Reply:
Those changes were needed after i run pylint checks. I will check if still needed.